scipy/stats/tests/test_multivariate.py

```diff
@@ -2465,6 +2466,34 @@ def test_invalid_params(self):
                       [10, 15], 5.5)
 
 
+class TestConditionalTable:
+    def test_mean(self):
+        row = [2, 6]
+        col = [1, 3, 4]
+        m = conditional_table.mean(row, col)
+        assert_equal(np.sum(m), np.sum(row))
+
+    def test_rvs(self):
+        # test if `rvs` is unbiased and large sample size converges
+        # to the true mean.
+        row = [2, 6]
+        col = [1, 3, 4]
+        rvs = conditional_table.rvs(row, col, size=1000, random_state=123)
```

Comment:
Let's test the marginals directly. I think this will work.
```suggestion
        rvs = conditional_table.rvs(row, col, size=1000, random_state=123)
        assert_equal(rvs.sum(axis=-1), np.broadcast_to(row, (1000, 2)))
        assert_equal(rvs.sum(axis=-2), np.broadcast_to(col, (1000, 3)))
```

Reply:
ok

Reply:
Hah thanks. Doing SciPy more than is healthy for me pays off sometimes : P